lib/shared-class.js

```diff
@@ -137,7 +137,7 @@ SharedClass.prototype.methods = function(options) {
 SharedClass.prototype.isMethodEnabled = function(sharedMethod) {
   if (!sharedMethod.shared) return false;
 
-  var key = this.getKeyFromMethodNameAndTarget(sharedMethod.name, sharedMethod.isStatic);
+  var key = sharedMethod.name;
```

Comment:
This is not correct. The constructor of `SharedMethod` is still expecting name + `options.isStatic`. I am surprised this was not caught by unit-tests. Could you please check that we have unit-tests for `isMethodEnabled` for both static and prototype cases, and add them if they are missing?